wordpress-mobile/WordPress-iOS
# Jetpack Focus: Disable stats widgets in phase 4

Closes #19838

## Description
This PR disables the widgets during phase 4. Instead of displaying stats, we now display a message stating that Stats have moved.

### Screenshots

<img src="https://user-images.githubusercontent.com/25306722/214737455-0e436b44-a9ad-4a83-8161-8c4a7602d7df.png" width=50%>

## Testing Instructions

1. Open the debug menu and disable all "Jetpack Features Removal" flags
2. Add a widget to the home screen
3. The widget should display the selected site stats
4. Open the debug menu and enable the "Jetpack Features Removal Phase one" flag
5. Close and re-open the app
6. The app's UI should now be updated to not show JP features
7. Navigate to the home screen
8. Make sure the widget now displays a message stating stats have moved
9. Repeat the above steps for different widgets

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

5. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
